charts/incubator/typecho/values.yaml

```diff
@@ -0,0 +1,57 @@
+image:
+  repository: joyqi/typecho
+  tag: 1.2.0-php8.0-apache@sha256:ef5b30b60f0bb51321c51b99514e09b4aa012764d5ae2ce03abf6b6b7584b6e6
+  pullPolicy: IfNotPresent
+
+securityContext:
+  readOnlyRootFilesystem: false
+  runAsNonRoot: false
+
+podSecurityContext:
+  runAsUser: 0
+  runAsGroup: 0
+
+env:
+  MEMORY_LIMIT: "100M"
+  MAX_POST_BODY: "50M"
+  TYPECHO_INSTALL: "1"
```

Comment:
What happens if it is already installed and it starts again with this as `1`?

Reply:
1 is to automatically install the blog system, 0 you need to prepare your own files

Reply:
But what happens when the app restarts?

This flag will be again `1`. Will the app re'install the blog system?

Reply:
good question. I don't know.

Reply:
need to read its code

Reply:
I'd suggest, just to be safe.

We can set `TYPECHO_INSTALL` to `0` on values.yaml
and run an `installContainers` and set there the TYPECHO_INSTALL to 1.

So it will install the first time and then the flag will be 0 always.

I can fix that later (after work)

Reply:
OK

Reply:
They replied that the install will not be overwritten

Reply:
Awesome